README.md

```diff
@@ -1,51 +1,74 @@
 # Flow Cadence Template Utilities
 Working with Cadence template files can be a bit frustrating.
-We aim to simplify this process by providing you an easy way to generate supporting files, which are easy to use. 
+We aim to simplify this process by providing you an easy way to generate supporting JavaScript-wrapped template files, which are easy to use.
```

Comment:
Maybe something more like, "Flow Cadut supports Cadence interaction by wrapping template files in JavaScript."